### PR TITLE
add ovn-ic HA deploy

### PR DESCRIPTION
**Pull request by lut777**
Now ovn-ic db cluster is available when deploying cluster interconnection.

### CODE REVIEW

File: dist/images/start-ic-db.sh [hunk 1]
```diff
@@ -4,8 +4,51 @@ set -eo pipefail
 TS_NAME=${TS_NAME:-ts}
 TS_CIDR=${TS_CIDR:-169.254.100.0/24}
 
-/usr/share/ovn/scripts/ovn-ctl --db-ic-nb-create-insecure-remote=yes --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
-/usr/share/ovn/scripts/ovn-ctl status_ic_ovsdb
-ovn-ic-nbctl --may-exist ts-add "$TS_NAME"
-ovn-ic-nbctl set Transit_Switch ts external_ids:subnet="$TS_CIDR"
-tail -f /var/log/ovn/ovsdb-server-ic-nb.log
+rm -f /etc/ovn/ovn_ic_nb_db.db /etc/ovn/ovn_ic_sb_db.db /var/log/ovn/ovsdb-server-ic-nb.log /var/log/ovn/ovsdb-server-ic-sb.log
```
Comment (oilbeater) [inline dist/images/start-ic-db.sh:7]: container restart will delete the db and lost all data

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: docs/cluster-interconnection.md [hunk 2]
```diff
@@ -147,7 +147,32 @@ In az2
...
+1. Run the leader Interconnection Controller in a region that can be accessed by other cluster.  
+
+   `LEADERIP` is the IP of node where leader controller is deployed.
+
+   `NODE_IPS` is the IP of all nodes in the interconnection Controller cluster, in the format `IP1,IP2,...,IPn`.
+
+   `LOCALIP` is the IP of the node on which the container will run. For the leader, `LOCALIP` is its own IP.
+
+```bash
+docker run --name=ovn-ic-db -d --network=host -v /etc/ovn/:/etc/ovn -v /var/run/ovn:/var/run/ovn -v /var/log/ovn:/var/log/ovn -e LOCAL_IP="LEADERIP"  -e NODE_IPS="IP1,IP2,IP3"   kubeovn/kube-ovn:v1.9.0 bash start-ic-db.sh
```
Comment (oilbeater) [inline docs/cluster-interconnection.md:163]: containerd command is also required

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Review by oilbeater — APPROVED** [review]

File: dist/images/start-ic-db.sh (hunk 1 above)
Comment (lut777) [reply dist/images/start-ic-db.sh]: deleting db is deleted now.

File: docs/cluster-interconnection.md (hunk 2 above)
Comment (lut777) [reply docs/cluster-interconnection.md:163]: added, along with HA ovn-ic config.

**Pull request merged**